cpu/kinetis_common/include/periph_cpu.h

```diff
@@ -166,6 +166,14 @@ typedef struct {
  */
 void gpio_init_port(gpio_t pin, uint32_t pcr);
 
+/**
+ * @brief Override uart_t
+ */
+#define HAVE_UART_T
+typedef uint8_t uart_t;
+#define UART_DEV(x)         (x)
+#define UART_UNDEF          0xff
```

Comment:
I honestly don't remember what was the reasoning behind it.. I will remove it for now, let's clean up the driver in a separate PR.